src/effects/builtin/reverb/ReverbView.qml

```diff
@@ -1,20 +1,104 @@
 import QtQuick 2.15
 
+import Muse.UiComponents
+
 import Audacity.Effects
 
 import "../common"
 
 EffectBase {
 
-    property string title: reverb.title
+
     property alias instanceId: reverb.instanceId
 
-    width: 300
-    height: 200
+    property string title: qsTrc("effects", "Reverb")
+    property bool isApplyAllowed: true
+
+    height: 400
+    width: 400
 
     ReverbViewModel {
         id: reverb
     }
 
+    Component.onCompleted: {
+        reverb.init()
+    }
+
+    component ParamItem: Item {
+        id: param
+        property alias title: label.text
+        property real value: 0.0
+        property alias min: slider.from
+        property alias max: slider.to
 
+        height: 40
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.margins: 8
+
+        StyledTextLabel {
+            id: label
+            anchors.left: parent.left
+            anchors.top: parent.top
```

Comment:
Would just `anchors.verticalCenter: parent.verticalCenter` work, like in `IncrementalPropertyControl` ?